### PR TITLE
v21: copy clusterapi v20

### PR DESCRIPTION
**Pull request by rossf7**

### CODE REVIEW

File: service/controller/clusterapi/v21/cluster_resource_set.go [hunk 1]
```diff
@@ -0,0 +1,453 @@
+package v20
```
Comment (sslavic) [inline service/controller/clusterapi/v21/cluster_resource_set.go]:
```suggestion
package v21
```

File: service/controller/clusterapi/v21/error.go [hunk 2]
```diff
@@ -0,0 +1,21 @@
+package v20
```
Comment (sslavic) [inline service/controller/clusterapi/v21/error.go]:
```suggestion
package v21
```

File: service/controller/clusterapi/v21/machine_deployment_resource_set.go [hunk 3]
```diff
@@ -0,0 +1,152 @@
+package v20
```
Comment (sslavic) [inline service/controller/clusterapi/v21/machine_deployment_resource_set.go]:
```suggestion
package v21
```

File: service/controller/clusterapi/v21/cluster_resource_set.go (hunk 1 above)
Comment (rossf7) [reply service/controller/clusterapi/v21/cluster_resource_set.go]: This is intentional to make the PRs easier to remove. Imports will be updated in later PRs.

https://github.com/giantswarm/giantswarm/blob/master/processes/new-version-bundle.md

**Pull request merged**